src/pages/Dashboard.vue

```diff
@@ -97,6 +97,7 @@ import StatusPanel from '@/components/panels/StatusPanel.vue'
 import ToolsPanel from '@/components/panels/ToolsPanel.vue'
 import WebcamPanel from '@/components/panels/WebcamPanel.vue'
 import ZoffsetPanel from '@/components/panels/ZoffsetPanel.vue'
+import _ from 'lodash'
```

Comment:
Due to bundle size, it's better if we'd use `lodash.kebabcase` as package. This is much smaller than the full `lodash` package.

Reply:
thx and fixed it!